docs/src/pages/components/material-icons/SearchIcons.js

```diff
@@ -483,11 +487,17 @@ export default function SearchIcons() {
     [],
   );
 
+  const handleQueryChange = React.useCallback(
```

Comment:
Do we need the `React.useCallback` on this one? The prop seems to be provided to the component directly
```suggestion
  const handleQueryChange = 
```

Reply:
Since I don't necessarily know the inner workings of a component (it may memoize on that property), I made a habit of making sure to pass referentially stable values whenever I can. 
I don't mind removing it, it probably doesn't matter much here.